bitcoin/tx.h

```diff
@@ -12,6 +12,10 @@
 
 #define BITCOIN_TX_DEFAULT_SEQUENCE 0xFFFFFFFF
 
+struct witscript {
+    u8 *ptr;
+};
+
```

Comment:
In many other locations we just hand a `u8 *` around for this. Is your goal to have more type-checking here?

Reply:
I'm trying to marshal a vector of witness scripts in the wire message:
https://github.com/ksedgwic/lightning-1/blob/33d756bf6a65c5fa8b1d11b5233029bd2f49a950/hsmd/hsm_wire.csv#L161-L162
I don't see how to specify an array of length-prefixed u8* should be serialized without the wrapping struct.
Is there a better way?

Reply:
ping?